contracts/Manager.sol

```diff
@@ -508,6 +661,79 @@ contract Manager is UUPSOwnableUpgradeable, UsingRegistryUpgradeable {
         account.scheduleWithdrawals(beneficiary, finalGroups, finalVotes);
     }
 
+    /**
+     * @notice Used to withdraw CELO from the system from specific validator group
+     * that account voted for previously.
+     * @param specificGroup The specific validator group that we want to withdraw from.
+     * @param withdrawal The amount of stCELO to withdraw.
+     * @return groups The groups to withdraw from.
+     * @return votes The amount to withdraw from each group.
+     */
+    function withdrawFromSpecificGroup(
+        address specificGroup,
+        uint256 withdrawal,
+        uint256 stCeloAmount
+    ) private returns (address[] memory groups, uint256[] memory votes) {
+        uint256 votesRemaining = account.getCeloForGroup(specificGroup);
+        uint256 votesToWithdraw = Math.min(votesRemaining, withdrawal);
```

Comment:
Should we continue if we aren't able to withdraw the desired amount?

Reply:
With introduction of rebalancing it will now revert